packages/next/src/withPayload.js

```diff
@@ -129,45 +118,14 @@ export const withPayload = (nextConfig = {}, options = {}) => {
 
       return {
         ...incomingWebpackConfig,
-        externals: [
-          ...(incomingWebpackConfig?.externals || []),
-          'drizzle-kit',
-          'drizzle-kit/api',
-          'sharp',
-          'libsql',
-          'require-in-the-middle',
-        ],
-        ignoreWarnings: [
-          ...(incomingWebpackConfig?.ignoreWarnings || []),
-          { module: /node_modules\/mongodb\/lib\/utils\.js/ },
-          { file: /node_modules\/mongodb\/lib\/utils\.js/ },
-          { module: /node_modules\/mongodb\/lib\/bson\.js/ },
-          { file: /node_modules\/mongodb\/lib\/bson\.js/ },
-        ],
+        externals: [...(incomingWebpackConfig?.externals || []), 'require-in-the-middle'],
         plugins: [
           ...(incomingWebpackConfig?.plugins || []),
           // Fix cloudflare:sockets error: https://github.com/vercel/next.js/discussions/50177
           new webpackOptions.webpack.IgnorePlugin({
             resourceRegExp: /^pg-native$|^cloudflare:sockets$/,
           }),
         ],
-        resolve: {
-          ...(incomingWebpackConfig?.resolve || {}),
-          alias: {
-            ...(incomingWebpackConfig?.resolve?.alias || {}),
-          },
-          fallback: {
-            ...(incomingWebpackConfig?.resolve?.fallback || {}),
-            '@aws-sdk/credential-providers': false,
```

Comment:
This is a relict as well. With our clean server-client separation, those dependencies shouldn't sneak into the client bundle anyways.